components/core/src/clp/CommandLineArguments.hpp

```
@@ -22,9 +22,17 @@ namespace clp {
         };
 
         // Constructors
-        explicit CommandLineArguments (const std::string& program_name) : CommandLineArgumentsBase(program_name), m_show_progress(false),
-                m_print_archive_stats_progress(false), m_target_segment_uncompressed_size(1L * 1024 * 1024 * 1024),
-                m_target_encoded_file_size(512L * 1024 * 1024), m_target_data_size_of_dictionaries(100L * 1024 * 1024), m_compression_level(3) {}
+        explicit CommandLineArguments (const std::string& program_name) :
+                CommandLineArgumentsBase(program_name),
+                m_show_progress(false),
+                m_print_archive_stats_progress(false),
+                m_use_glt(false),
+                m_target_segment_uncompressed_size(1L * 1024 * 1024 * 1024),
+                m_target_encoded_file_size(512L * 1024 * 1024),
+                m_target_data_size_of_dictionaries(100L * 1024 * 1024),
+                m_compression_level(3),
+                m_glt_combine_threshold(std::numeric_limits<double>::max())
```

This should be 0.1, right (the default)?

---

if we let the default to be 0.1, we won't be able to catch the case where user uses clp-flow but also manually assign a threshold to be 0.1 (clp should throw an error but we can't tell if the 0.1 comes from user or default)

That said, I feel we don't have to sacrifice code quality for such exterme case so I agreed with changing it to 0.1